table/policy.go

```diff
@@ -1090,6 +1088,14 @@ func NewExtCommunitySet(c config.ExtCommunitySet) (*ExtCommunitySet, error) {
 	}, nil
 }
 
+func (s *ExtCommunitySet) Append(arg DefinedSet) error {
```

Comment:
How about calling `s.regExpSet.Append(arg)` first, then append `subList` to `s.subtypeList`?
`regExpSet.Append()` is checking `arg`'s type. I think it is useful to check that here.